job/label_sync_cronjob.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1beta1
+kind: CronJob
```

Comment:
Just like to confirm, all the configurations are exact same with on-demand jobs, right? 
https://github.com/kitex-contrib/test-infra/blob/main/job/member_sync_job.yaml

Reply:
Yes